### ISSUE TITLE
Loadbalancer gets created but targets are empty

### ISSUE THREAD

**Issue by tibeer** [hetznercloud/hcloud-cloud-controller-manager#463]
Hi there!

I'm testing around with the cloud controller manager and noticed that I get a loadblanacer provisioned trough an ingress resource, though, it has not targets. I guess there is an annotation or something missing?

Infrastructure layer:

1 VM with two network interfaces: 1 public IPv4, 1 in a private network (10.0.0.0/8, subnet is 10.0.0.0/16)

Kubernetes layer:

```sh
kubeadm init --pod-network-cidr=10.244.0.0/16
kubectl apply -f https://github.com/flannel-io/flannel/releases/latest/download/kube-flannel.yml
kubectl -n kube-flannel patch ds kube-flannel-ds --type json -p '[{"op":"add","path":"/spec/template/spec/tolerations/-","value":{"key":"node.cloudprovider.kubernetes.io/uninitialized","value":"true","effect":"NoSchedule"}}]'
kubectl -n kube-system create secret generic hcloud --from-literal=token=MYTOKEN --from-literal=network=MYNETWORKNAME
helm repo add hcloud https://charts.hetzner.cloud
helm repo update
helm install hccm hcloud/hcloud-cloud-controller-manager -n kube-system --set networking.enabled=true
```

App layer:

```sh
helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
helm repo update
helm install ingress-nginx ingress-nginx/ingress-nginx --namespace ingress-nginx --create-namespace --set controller.service.annotations."load-balancer\.hetzner\.cloud/location"="fsn1"
```

The ingress / service of that ingerss gets both IPv4 and IPv6 addresses assigned, though when I look in the Hetzner Website Dashboard, I can see that the newly created loadbalancer does not have any targets. Maybe you can help me here.
Kind regards Tim

**Comment by tibeer** [comment]
Maybe another hint: My kubernetes api endpoints take the first NIC which is the public IP. Maybe this might be a reason, why the LB does not pick up the server as a target?

**Comment by tibeer** [comment]
Never mind. I found the solution: Having only one node deployed and no workers is the reason. The Control Plane node automatically get's these labels:

```yaml
    node-role.kubernetes.io/control-plane: ""
    node.kubernetes.io/exclude-from-external-load-balancers: ""
```

Once removed, the Loadbalancer created from the CCM adds this node like expected to the targets. Though, I'm not sure if this will also happen after the deployment of CCM. In my case, I removed the flags right after `kubeadm init`. Works now :)
